src/generic/grid.cpp

```diff
@@ -1871,6 +1871,7 @@ void wxGrid::Render( wxDC& dc,
     dc.DrawRectangle( pointOffSet, sizeCells );
 
     // draw cells
+    wxDCClipper clipper( dc, wxRect( pointOffSet, sizeCells ) );
```

Comment:
Hmm, this looks wrong to me. The [original patch](http://trac.wxwidgets.org/attachment/ticket/17872/gridrenderclipping.patch) had a block here and I think it was correct, we don't want this clipping rectangle to affect the other calls below, it should only apply to drawing the grid cell area.

So I'm changing this, please let me know if you disagree.